docs/src/gui/qtvcp.adoc

```diff
@@ -669,176 +868,208 @@ def get_handlers(halcomp,widgets,paths):
-It's usually better to use the special closing_cleanup__ function.
+Putting the *`closeEvent` function here will catch closing events*.
+
+This _replaces any predefined `closeEvent`_ function from QtVCP.
 
 [source,python]
 ----
-    def closeEvent(self, event):
-        self.do_something()
-        event.accept()
+def closeEvent(self, event):
+    self.do_something()
+    event.accept()
 ----
 
-== Connecting widgets to python code
+NOTE: It is usually better to use the special `closing_cleanup__`
+  function.
+
+== Connecting Widgets To Python Code
 
-It's possible to connect widgets to python code using signals and slots.
-In this way you can give new functions to linuxcnc widgets or utilize
-standard widgets to control linuxcnc.
+It is possible to connect widgets to Python code using *signals and slots*.
+
+In this way you can:
+
+* _Give new functions to LinuxCNC widgets_, or
+* _Utilize standard Qt widgets to control LinuxCNC_.
 
 === Overview
 
-In the Designer editor you would create user function slots and connect
-them to widgets using signals.
-In the handler file you would create the slot's functions defined in Designer.
-
-[[cha:designer-slots]]
-=== Using Designer to add slots
-
-When you have loaded your screen into designer add a plain PushButton to the screen.
-You could change the name of the button to something interesting like 'test_button'
-There are two ways to edit connections - This is the graphical way
-There is a button in the top tool bar of designer for editing signals.
-After pushing it, if you click-and-hold on the button it will show a arrow
-(looks like a ground signal from electrical schematic)
-Slide this arrow to a part of the main window that does not have widgets on it.
-A 'Configure Connections' dialog will pop up.
-The list on the left are the available signals from the widget.
-The list on the right is the available slots on the main window and you can add to it.
-
-Pick the signal 'clicked()' - this makes the slots side available.
-click 'edit' on the slots list.
-A 'Slots/Signals of MainWindow' dialog will pop up.
-On the slots list at the top there is a plus icon - click it.
-you can now edit a new slot name.
-Erase the default name 'slot()' and change it to test_button()
-press the ok button.
-You'll be back to the 'Configure Connections' dialog.
-now you can select your new slot in the slot list.
-then press ok and save the file.
-
-.Designer signal/slot selection
-image::images/designer_slots.png["QTvcp",align="left"]
-
-=== Handler file changes
-
-Now you must add the function to the handler file.
-The function signature is 'def slotname(self):'
-We will add some code to print the widget name.
-
-So for our example:
+*In the Qt Designer editor*:
+
+* You _create user function slots_
+* You _connect the slots to widgets using signals_.
+
+*In the handler file*:
+
+* You _create the slot's functions_ defined in Qt Designer.
+
+[[sub:qtvcp:designer-slots]]
+=== Using Qt Designer To Add Slots
```

Comment:
```suggestion
=== Using Qt Designer to Add Slots
```
or
```suggestion
=== Using Qt Designer to add Slots
```
? 
I am not sure.

Reply:
capitalized (even though I dislike this for level three titles)

Reply:
As said in previous comment, reading the excellent GitLab style guide and referenced documents makes me believe we should get away from camel capitalized titles and settle on sentence capitalization.
What remains less clear to me is the capitalization of "special" words, like "Slots" in this case...

Reply:
Reading again https://docs.gitlab.com/ee/development/documentation/styleguide/#feature-names I believe "Slots" should indeed be title cased.